test/test-utils.tsx

```diff
@@ -1,8 +1,13 @@
 import { render } from '@testing-library/react'
+import type { ReactNode } from 'react'
 import type { RenderOptions, RenderResult } from '@testing-library/react'
 
-const Providers = ({ children }) => {
-  return children
+export type ProvidersProps = {
+  children?: ReactNode
```

Comment:
Something to note is that we have been accessing the React types via the namespace so far in this project (e.g. `React.ReactNode`). Though maybe we want to reconsider this as we have been explicit about this in regard to the other types. Thoughts @chasm?

Reply:
Not sure what you mean, @reubenberghan . What are you proposing? The materials I've read recommend using ReactNode for children. I wouldn't want to define that myself, so I'm fine with importing it. Could import it as a type, I guess. But I'm not worried about the React namespace, as long as we don't introduce our own namespaces.

That said, hmm.  We could be more specific -- ReactNode works like this, I think:

```ts
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined
```

We could decide that the children will never be null or undefined or a boolean or a fragment and just type them as ReactChild, for example. But I think that might be overkill. I'd say leave it.

Reply:
Sorry I'm basically talking about whether we explicitly import the React types vs accessing them via the namespace so for example:
```
import type { ReactNode } from 'react'

type ProviderProps = {
  children?: ReactNode
}
```
Versus:
```
type ProviderProps = {
  children?: React.ReactNode
}
```

Reply:
Ah, I see. Yes, I am in favor of the first option. Import types specifically as types. They are still in the React namespace, of course (thus my confusion), but we're bringing them in to the module namespace. We can alias if need be.

In general, I find having to use the namespace to be a lot of noise in the code. You can always look to the imports to see what namespace they came from. Also, having everything mentioned in the imports means that first thing on the page you can see everything it depends on from outside the module.

Reply:
@natebowang change this to @reubenberghan 's version and we're good to go.

Reply:
@chasm Sorry, I am confused. Do you mean to remove types and change this file back to `.js`.

Reply:
@chasm If we are talking about ReactNode type, it is import as a type now:
```
import type { ReactNode } from 'react' 

type ProviderProps = { 
  children?: ReactNode 
}
```

Reply:
No. All good now. Do not go back to .js.